src/Engine3D.ts

```diff
@@ -281,44 +313,108 @@ export class Engine3D {
         this.render(0);
     }
 
+    /**
+     * get view render job instance
+     * @param view 
+     * @returns 
+     */
     public static getRenderJob(view: View3D): RendererJob {
         return this.renderJobs.get(view);
     }
 
+    /**
+     * Pause the engine render
+     */
+    public static pause() {
+        requestAnimationFrame(null);
```

Comment:
I think this will not cancel the previous loop

Reply:
this bug fix in my pr #99